pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -1550,15 +1557,15 @@ public void testMetadataStoreStats() throws Exception {
         String metricsStr = output.toString();
         Multimap<String, Metric> metricsMap = parseMetrics(metricsStr);
 
-        Collection<Metric> getOpsFailed = metricsMap.get("pulsar_metadata_store_get_ops_failed" + "_total");
-        Collection<Metric> delOpsFailed = metricsMap.get("pulsar_metadata_store_del_ops_failed" + "_total");
-        Collection<Metric> putOpsFailed = metricsMap.get("pulsar_metadata_store_put_ops_failed" + "_total");
+        Collection<Metric> getOpsFailed = metricsMap.get("pulsar_metadata_store_get_failed" + "_total");
```

Comment:
Out of curiosity, while the `+` in the middle? Usually when you have the metric name you CMD-F and search. That plus sign will only get in the way :)

Reply:
`_total` added by Prometheus, use `+` to distinguish